.github/workflows/linux-build.yml

```diff
@@ -55,5 +55,5 @@ jobs:
       run: sudo apt install -y g++-11
 
```

Comment:
Since you we don't need to specify the `buildSystem `and the `architecture `anymore, I think we should remove them from matrix,